pkg/internal/checkup/executor/console/login.go

```diff
@@ -29,14 +29,13 @@ import (
 	"google.golang.org/grpc/codes"
 )
 
-func LoginToCentOS(serialConsoleClient vmiSerialConsoleClient, vmiNamespace, vmiName, username, password string) error {
+func (e Expecter) LoginToCentOS(username, password string) error {
 	const (
 		connectionTimeout = 10 * time.Second
 		promptTimeout     = 5 * time.Second
 	)
 
-	expecter := NewExpecter(serialConsoleClient, vmiNamespace, vmiName)
-	genExpect, err := expecter.spawnConsole(connectionTimeout)
+	genExpect, err := e.spawnConsole(connectionTimeout)
```

Comment:
Could you please elaborate on the meaning of `genExpect`?

Reply:
Added an explanation on the 3rd commit where it was first introduced.